tests/lwe/ops.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: heir-opt --split-input-file %s 2>&1 | FileCheck %s
```

Comment:
Consider adding an invalid case, using the diagnostic infrastructure to assert the error message:

See, e.g., https://github.com/google/heir/blob/53124b671d1152b23fd44818de00231dbae13273/tests/secret/verifier.mlir#L1-L4

`--verify-diagnostics` enables the checks, and ` // expected-error@+1 {{<regex>}}` checks for the error emitted at the op below the comment.

Not sure if it needs to be in a separate file due to FileCheck complaining about no input after the diagnostic check passes...

Reply:
Added some invalid cases! Also lead me to write some verifiers ensuring that the encoding and lwe params op attrs match the outputs' attributes.